builder.go

```diff
@@ -175,7 +176,7 @@ func (b *Builder) Local(v string, expr *Expression) *Builder {
 		return b
 	}
 
-	b.locals = append(b.locals, local{variable: variable(v), expr: expr})
+	b.locals = append(b.locals, local{variable: variable(varSymbol + v), expr: expr})
```

Comment:
i thought we quit on adding prefixes on variables, and literals
```suggestion
	b.locals = append(b.locals, local{variable: variable(v), expr: expr})
```

Reply:
Yes, you are right, forgot to remove it ;)